pkgs/applications/audio/sidplayfp/default.nix

```diff
@@ -9,32 +11,33 @@
 , libpulseaudio
 }:
 
-assert alsaSupport -> alsaLib != null;
-assert pulseSupport -> libpulseaudio != null;
 let
   inherit (lib) optional;
-  inherit (lib.versions) majorMinor;
 in
 stdenv.mkDerivation rec {
   pname = "sidplayfp";
-  version = "2.0.2";
+  version = "2.1.1";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/sidplay-residfp/sidplayfp/${majorMinor version}/${pname}-${version}.tar.gz";
-    sha256 = "1s2dfs9z1hwarpfzawg11wax9nh0zcqx4cafwq7iysckyg4scz4k";
+  src = fetchFromGitHub {
+    owner = "libsidplayfp";
+    repo = "sidplayfp";
+    rev = "v${version}";
+    sha256 = "0s3xmg3yzfqbsnlh2y46w7b5jim5zq7mshs6hx03q8wdr75cvwh4";
   };
 
-  nativeBuildInputs = [ pkg-config ]
+  nativeBuildInputs = [ autoreconfHook perl pkg-config ]
     ++ optional alsaSupport alsaLib
     ++ optional pulseSupport libpulseaudio;
 
   buildInputs = [ libsidplayfp ];
 
+  enableParallelBuilding = true;
+
   meta = with lib; {
     description = "A SID player using libsidplayfp";
-    homepage = "https://sourceforge.net/projects/sidplay-residfp/";
+    homepage = "https://github.com/libsidplayfp/sidplayfp";
     license = with licenses; [ gpl2Plus ];
-    maintainers = with maintainers; [ dezgeg ];
-    platforms = with platforms; linux;
+    maintainers = with maintainers; [ dezgeg OPNA2608 ];
+    platforms = with platforms; linux ++ windows;
```

Comment:
```suggestion
    platforms = platforms.all;
```

Reply:
I'm inclined to disagree with it supporting Darwin, because I can't find an actual CoreAudio driver in [the source tree](https://github.com/libsidplayfp/sidplayfp/tree/v2.1.1/src/audio) and without audio output, as an audio player, *what's the point*… But I guess it technically may compile on that platform and fall back to a dummy / recording-only mode.

Reply:
platforms should be what upstream supports and broken should be set if it does not work on some plaform.

Reply:
I know how `platforms` and `broken` work, now you're just coming off as rude…

I'll add some WIP commits to test if/how well it works, I'll let you know when it's review-ready again.